From 91a31bd9fb0bdf8332d1a6803d1e5cdc201807b5 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Fri, 9 Jul 2010 23:48:36 +0800
Subject: [PATCH] ENGR00124956 Increase vpu iram size to 80K

Increase vpu iram size to 80K on mx51 and mx53 to
use more iram for video decoding and encoding.
It can cover 720P encoder and 1080P decoder except VC-1 AP.

Signed-off-by: Sammy He <r62914@freescale.com>w
---
 arch/arm/plat-mxc/include/mach/mx5x.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mx5x.h b/arch/arm/plat-mxc/include/mach/mx5x.h
index 3b9728e..879a085 100644
--- a/arch/arm/plat-mxc/include/mach/mx5x.h
+++ b/arch/arm/plat-mxc/include/mach/mx5x.h
@@ -93,7 +93,7 @@
 #endif
 
 #ifdef CONFIG_MXC_VPU_IRAM
-#define VPU_IRAM_SIZE  0x11000
+#define VPU_IRAM_SIZE  0x14000
 #else
 #define VPU_IRAM_SIZE 0
 #endif
-- 
1.5.4.4

